qa/TL0_python-self-test-operators_1/test_body.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash -e
 
 test_py_with_framework() {
-    ${python_new_invoke_test} -A '!slow' -s operator_1
+    ${python_new_invoke_test} -s operator_1
```

Comment:
this `-A '!slow'` didn't do anything

Reply:
Mhhh
I was able to run this simple example:
```
from nose.plugins.attrib import attr


def test1():
    pass

@attr("slow")
def test2():
    pass
```
with `python3 -m nose2 --verbose -A '!slow' -s test` and `python3 -m nose2 --verbose -A 'slow' -s test` and it provided the expected results.

I think now we just rely on `unittest_slow.cfg` which defines the `slow_` test prefix instead of using the attribute.

Reply:
Ok, it doesn't work with `yield`, only with `params`. I'm fine either way.